src/fides/api/ctl/database/seed.py

```diff
@@ -0,0 +1,263 @@
+"""
+Provides functions that seed the application with data.
+"""
+from typing import List
+from fideslang import DEFAULT_TAXONOMY
+from fideslib.exceptions import KeyOrNameAlreadyExists
+from fideslib.models.client import ClientDetail
+from fideslib.utils.text import to_snake_case
+from loguru import logger as log
+
+from fides.api.ctl.database.session import sync_session
+from fides.api.ctl.sql_models import sql_model_map
+from fides.api.ctl.utils.errors import AlreadyExistsError, QueryError
+from fides.api.ops.models.policy import ActionType, DrpAction, Policy, Rule, RuleTarget
+from fides.api.ops.models.storage import StorageConfig
+from fides.api.ops.schemas.storage.storage import (
+    FileNaming,
+    ResponseFormat,
+    StorageDetails,
+    StorageType,
+)
+from fides.ctl.core.config import get_config
+
+from .crud import create_resource, list_resource
+
+CONFIG = get_config()
+FIDESOPS_AUTOGENERATED_CLIENT_KEY = "fides_autogenerated_client"
+FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+STRING_REWRITE_STRATEGY_NAME = "string_rewrite"
+ERASURE_POLICY_KEY = "default_erasure_policy"
+DEFAULT_ACCESS_POLICY = "default_access_policy"
+DEFAULT_ACCESS_POLICY_RULE = "default_access_policy_rule"
+DEFAULT_ERASURE_POLICY_RULE = "default_erasure_policy_rule"
+
+
+def filter_data_categories(
```

Comment:
@NevilleS turns out this was way more complex to implement than I expected, but it works now and is well-tested for the future

by looking at this function, you can see how much actual filtering had to happen here to prevent breakage and match what was hard-coded before

Reply:
Yeah- understood. Thanks for doing that, it makes this maintainable going forward.